jobs/build-webapp.groovy

```diff
@@ -553,6 +552,12 @@ def deployAndReport() {
       }
       parallel(jobs);
 
+      // Run static build and deploy by itself. This is to avoid intermittent
+      // OOM issues during the rspack i18n build.
+      parallel([
+         "deploy-to-gcs": { deployToGCS(); }
```

Comment:
Another option, either instead of or in addition to this, would be to run `ulimit` in the same shell as the rspack build, to limit the amount of memory rspack could use.  If we use `ulimit -H`, it would make rspack crash if it tried to use more memory, which I think is what we want.